src/__tests__/Search.test.tsx

```diff
@@ -19,6 +15,10 @@ import { FilterLanguagesCommonTests } from "./componentHelpers/FilterLanguages";
 import { FilterFormatTests } from "./componentHelpers/FilterFormats";
 import { findFiltersForField } from "../util/SearchQueryUtils";
 import filterFields from "../constants/filters";
+import mockRouter from "next-router-mock";
+
+jest.mock("next/router", () => require("next-router-mock"));
```

Comment:
is this how we are mocking the next/router? just curious; wasn't sure what this line was doing but seems to be working

Reply:
Yes - we used to use `MockNextRouter.tsx` but it was hacked together, so using an actual library seemed more maintainable